### DISCUSSION TITLE
nginx ingress version

### DISCUSSION THREAD

**Discussion by hariprasadiit** [kube-hetzner/terraform-hcloud-kube-hetzner#304] [Q&A] [answered]
Hello,

Thanks for the awesome project.

I've created a cluster recently by enabling `nginx`. From the helm releases, I can see that the chart version is`0.14` and it's not `ingress-nginx` but `nginx-ingress` chart. 

is `nginx-ingress` is the intended chart here? If so, reference mentioned in the example nginx values file seems to be pointing to kubernetes maintained version.

https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml

**Comment by mysticaltech** [comment, marked as answer]
Good catch! I opened #305.
